sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml

```diff
@@ -6,6 +6,8 @@
   <arg name="map_calibration_area_file" default="map_calibration_area.pcd"/>
   <arg name="use_calibration_area_map" default="true" />
   <arg name="calibration_rviz" default="true"/>
+  <arg name="fitness_score_threshold" />
```

Comment:
aip_xx1 is outside of our project. Do we have to fix it too?

Reply:
I reached [agreement](https://star4.slack.com/archives/C04CKSQ7ZFZ/p1696230427088419?thread_ts=1695974107.937419&cid=C04CKSQ7ZFZ) with our project to apply the changes only to aip_x1 and aip_x1_1, not aip_xx1.
Therefore I will revert the aip_xx1 changes.

Reply:
I fixed at https://github.com/tier4/CalibrationTools/pull/129/commits/8db247c45692f903fb13a255e03a150dadfcd1e5 , https://github.com/tier4/CalibrationTools/pull/129/commits/12865fa8e8e88eca7c5a5086120ee1c84ad474c2 , and https://github.com/tier4/CalibrationTools/pull/129/commits/373c039ba61b3026c4bd52c89887100a2822bd0f .